sounddevice.py

```diff
@@ -2262,13 +2262,68 @@ def reset(self):
         vars(self).clear()
         self.__init__()
 
+
 if not hasattr(_ffi, 'I_AM_FAKE'):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
 
 
 class PortAudioError(Exception):
-    """This exception will be raised on PortAudio errors."""
+    """This exception will be raised on PortAudio errors.
+
+    Parameters
+    ----------
+    msg : str, optional
+        User error message.
+    code : int, optional
+        PaErrorCode value or None if there is no PaErrorCode associated
+        with the error.
```

Comment:
I would probably not explicitly discourage its use, but I simply cannot imagine a situation where a "normal" user would like to raise that exception. Can you think of one?

If somebody wants to implement yet unavailable PortAudio calls, yeah why not, they can use `PortAudioError`, but they sure don't need the docstring to tell them how to instantiate this exception.